fusion-cli/test/e2e/split-translations/fixture/src/split1.js

```diff
@@ -2,9 +2,11 @@
 
 import React from 'react';
 import {Translate} from 'fusion-plugin-i18n-react';
+import OtherComponent from './other-component.js';
```

Comment:
This seems a little crazy that importing another component would break the build, but it does. It's possible this becomes a different type of `AsyncBlock` dependency like a `ConcatenatedModule` or something when importing code that doesn't exist in a different bundle.

On master:
The module id in the instrumentation is an absolute path to a file, but the module id is the hash `fPq8`.

On this branch:
The module id in the instrumentation and the chunk are both `__fusion__0`.